onnxruntime/test/util/include/default_providers.h

```diff
@@ -36,8 +36,9 @@ namespace test {
 // unique_ptr providers with default values for session registration
 std::unique_ptr<IExecutionProvider> DefaultCpuExecutionProvider(bool enable_arena = true);
 std::unique_ptr<IExecutionProvider> DefaultCudaExecutionProvider();
-std::unique_ptr<IExecutionProvider> DefaultDnnlExecutionProvider(bool enable_arena = true);
-//std::unique_ptr<IExecutionProvider> DefaultTvmExecutionProvider();
+std::unique_ptr<IExecutionProvider> DefaultDnnlExecutionProvider();
+std::unique_ptr<IExecutionProvider> DnnlExecutionProviderWithOptions(const OrtDnnlProviderOptions* params);  
+// std::unique_ptr<IExecutionProvider> DefaultTvmExecutionProvider();
```

Comment:
It looks like this line had been removed. Did we re-add it when rebasing.

Should `// std::unique_ptr<IExecutionProvider> DefaultTvmExecutionProvider();` be removed?

Reply:
I think a space was added automatically and that's why the line appears as if it was removed and we added it, but it is on main branch as a comment so I don't want to remove it. I'll remove the extra space

Reply:
thanks for checking